elt-pipelines/facility_ops/ingest/beamlines/moderator_performance/fit_monitor.py

```
@@ -0,0 +1,180 @@
+        amplitude, peak_centre, sigma = popt
+        amplitude_err, peak_centre_err, sigma_err = perr
+        return MonitorPeak(
+            monitor_ws.run_info,
+            float(peak_centre),
+            float(peak_centre_err),
+            float(amplitude),
+            float(amplitude_err),
+            float(sigma),
+            float(sigma_err),
+        )
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)